src/main/java/net/greghaines/jesque/worker/WorkerImpl.java

```diff
@@ -558,6 +564,22 @@ protected void recoverFromException(final String curQueue, final Exception ex) {
         }
     }
 
+    private RecoveryStrategy getRecoveryStrategy(final String curQueue, final Exception ex) {
+        RecoveryStrategy recoveryStrategy;
+
+        if(ex instanceof JedisNoScriptException){
+            try {
+                loadScripts();
```

Comment:
getRecoveryStrategy is loading scripts? This is weird and not expected.

Reply:
Agree, my fault